assets/templates/dining_lists/dining_day.html

```diff
@@ -1,37 +1,22 @@
 {% extends 'base.html' %}
 {% load staticfiles %}
+{% load humanize %}
 
 {% block title %}Scala Dining {{ date }} {% endblock %}
 
 {% block content %}
     <div class="col-12 row p-0 m-0">
-        {% if prev_link %}
-            <a href="{{ prev_link }}" class="col-2 btn btn-block btn-blank">&#x25C0;</a>
-        {% else %}
-            <a class="col-2 btn btn-block btn-blank btn-disabled">&#x25C0;</a>
-        {% endif %}
-
-        {% if is_today %}
-            <div class="text-center col-8 ">
-            <b>{{ datename }}</b>
-        {% else %}
-            <div class="text-center col-8">
-            {{ datename }}
-        {% endif %}
-            </div>
-
-        {% if next_link %}
-            <a href="{{ next_link }}" class="col-2 btn btn-block btn-blank">&#x25B6;</a>
-        {% else %}
-            <a class="col-2 btn btn-block btn-disabled btn-blank">&#x25B6;</a>
-        {% endif %}
-    </div>
+        <a href="{% url 'day_view' day=previous_date.day month=previous_date.month year=previous_date.year %}"
+           class="col-2 btn btn-block btn-blank">&#x25C0;</a>
+
+        <div class="text-center col-8 ">
+            {{ date|naturalday|capfirst }}
```

Comment:
Het nadeel van 'today' en 'tomorrow' is dat ze foutief zijn zolang een pagina niet ververst is. Geen idee of we hier iets mee willen doen, maar ik wou het even kwijt.

Reply:
Ik neem aan dat je bedoelt dat ze niet meer kloppen als iemand de pagina meer dan een dag open laat staan? Om dit netjes te doen zou je dit client-side moeten renderen, maar dat doen we op het moment bijna nergens, dus laten we dat vooral niet hiervoor speciaal gaan doen. Misschien slim om een soort van page refresh te maken met een beetje javascript, zodat de pagina na een dag automatisch gaat herladen? Bijv.

```javascript
$(document).ready(function(){
    var page_load_date = Date.now();
    var page_reload_data = new Date(
        page_load_date.getFullYear(), 
        page_load_date.getMonth(),
        page_load_date.getDate() + 1
    );
    setTimeout(function() {
        location.reload(true); 
    }, page_reload_date - page_load_date);
});
```

Reply:
Ik vind dat we, tenzij we client side garanties kunnen geven, data het handigst als reservenet. Daarnaast mag er prima vandaag/morgen staan, maar dan tenminste vandaag maandag 10 dec. o.i.d.
Mbt optimalisatie van client side experiences, ik zou liever eerst alles statisch in orde hebben (zonder dat javascript o.i.d. wachttijd toe laat nemen) en dat we later gaan optimaliseren in dat soort ervaringen. Zelfde geld imo ook voor taal. Laten we eerst alles in het engels werkend maken, we kunnen altijd later taal implementeren (ipv halve implementatie waar engels en nederlands door elkaar gaat stromen.

Reply:
O. En dit zie ik net pas. Maar als je niet vandaag of morgen zit, heb je alleen een datum in jouw implementatie. Je wilt hier dag van de week hebben (i.e. woensdag 12 december) anders heb je een te groot risico dat mensen zich vergissen van dag.

Reply:
Oh ja dat is fout inderdaad, dat kan het beste met de `DATE_FORMAT` setting gefixt worden